mlos_bench/setup.py

```diff
@@ -76,7 +76,7 @@
         'mlos-core==' + _VERSION,
         'requests',
         'json5',
-        'jsonschema',
+        'jsonschema<4.18.0',    # FIXME: temporarily pinning to avoid breaking changes in 4.18.0
```

Comment:
This was causing CI issues due to the recently updated 4.18 release of jsonschema.  For now I pinned it back and will address it in #437

Reply:
PR for this is in #438 which should get merged first.